instance/tasks.py

```diff
@@ -66,6 +66,7 @@ def watch_pr():
             instance, created = OpenEdXInstance.objects.get_or_create(
                 sub_domain=pr_sub_domain,
                 fork_name=pr.fork_name,
+                github_pr_number=pr.number,
```

Comment:
@smarnach Since this is in a `get_or_create()` call, adding this will cause all existing PR sandbox to be duplicated when pushing this... Maybe add setting the PR number on the existing instances in the migration?